nat/traversal/pinger.go

```diff
@@ -352,7 +352,7 @@ func sortedConns(pings []pingResponse) []*net.UDPConn {
 
 // waitMsg waits until conn receives given message or timeouts.
 func waitMsg(ctx context.Context, conn *net.UDPConn, msg string) error {
-	ok := make(chan struct{})
```

Comment:
How does this help to fix a leak?

Reply:
It does not block on writing to channel when no one reads from the channel.

Reply:
Yes, good catch here.